app/models/school_data_importer.rb

```diff
@@ -24,18 +24,48 @@ def schools_data_file
   end
 
   def row_to_school(row)
+    local_authority = LocalAuthority.find_or_initialize_by(code: row.fetch("LA (code)"))
+    row_local_authority_name = row.fetch("LA (name)")
+    if local_authority.persisted? && local_authority.name != row_local_authority_name
+      Rails.logger.info "LA name change in school import. Old name: #{local_authority.name}, New name: #{row_local_authority_name}"
+    end
+    local_authority.name = row_local_authority_name
+    local_authority.save!
+
+    local_authority_district = LocalAuthorityDistrict.find_or_initialize_by(code: row.fetch("DistrictAdministrative (code)"))
+    row_lad_name = row.fetch("DistrictAdministrative (name)")
+    if local_authority_district.persisted? && local_authority_district.name != row_lad_name
```

Comment:
Spaces would add plenty of joy to devs reading the code at 3 am burning midnight oil

Reply:
Good point - I hope no dev has to read that at 3am, but if they do we might as well make it nice for them :)